src/dotnet-core-uninstall/MacOs/FileSystemExplorer.cs

```diff
@@ -46,12 +47,22 @@ public virtual IEnumerable<Bundle> GetAllInstalledBundles()
             return [..sdks, ..runtimes];
         }
 
-        private static bool IsMacx64Installation(string sdkVersionPath)
+        private static bool IsMacx64Installation(string bundlePath, bool isSdk)
         {
             try
             {
-                var rids = File.ReadAllText(Path.Combine(sdkVersionPath, "NETCoreSdkRuntimeIdentifierChain.txt"));
-                return !rids.Contains("osx-arm64");
+                if (isSdk)
+                {
+                    var rids = File.ReadAllText(Path.Combine(bundlePath, "NETCoreSdkRuntimeIdentifierChain.txt"));
+                    return !rids.Contains("osx-arm64");
```

Comment:
```suggestion
                    return rids.Contains("osx-x64");
```

Perhaps readability would be better if we avoid the double negative??